rest/client/request.go

```diff
@@ -103,6 +103,19 @@ func (c *communicatorImpl) createRequest(info requestInfo, data interface{}) (*h
 	return r, nil
 }
 
+func (c *communicatorImpl) bumpLastSentTime() {
```

Comment:
Redundant with implementations in rest/client/client.go.

Reply:
removed